_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -0,0 +1,318 @@
+---
+id: e88epe
+name: Image not in the accessibility tree is decorative
+rule_type: atomic
+description: |
+  This rule checks that visible img, svg and canvas elements that are ignored by assistive technologies are decorative
+accessibility_requirements:
+  wcag20:1.1.1: # Non-text content (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - Accessibiliy tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Wilco Fiers
+  assets:
+    - W3C
+    - Wikimedia
+    - Adobe
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+Any `img`, `canvas` or `svg` element that is [visible][] and for which one of the following is true:
+
+- **excluded**: The element is not [included in the accessibility tree][]; or
+- **ignored svg**: The element is an `svg` with an empty (`""`) [accessible name][] and a [semantic role][] of `graphics-document`; or
+- **ignored canvas**: The element is a `canvas` with an empty (`""`) [accessible name][] and no [explicit semantic role][]; or
+- **named from author**: The element is a [descendant][] in the [flat tree][] of an element that is [named from author][].
```

Comment:
Currently, images that are **named from author** ar applicable. This seems to not be what was intended (judging from Inapplicable example 4). If so, Applicability needs to be corrected.